app/js/views/element_call_button_view.js

```diff
@@ -0,0 +1,54 @@
+import { View } from 'components/fxos-mvc/dist/mvc';
+import 'components/gaia-button/gaia-button';
+
+(function(window) {
+'use strict';
+
+var shadowTemplate =
+`<style scoped>
+gaia-button {
+	--button-background: green;
+	--button-background-active: lightgreen;
+	--button-color: white;
+	--button-color-active: white;
+	width: 10rem;
+	min-width: 0 !important;
+	margin: 0.5rem !important;
+}
+</style>
+<gaia-button>Call</gaia-button>`;
+
+var proto;
+
+export default class ElementCallButtonView extends View {
+	constructor(controller, phoneFieldController) {
+		proto = Object.create(HTMLElement.prototype);
+		proto.createdCallback = this._createdCallback;
+		proto._scope = this;
```

Comment:
Hmm, so this is the scoping stuff you were talking about when using web components? I'm not sure if I like it, but would be interested in seeing what @mikehenrty and @justindarc have to say.

I'm leaning towards using a more classical MVC here instead of web components if it would avoid this scoping stuff. (The view layer can always be very slim and just proxy to a web component if we wanted).